src/lwc/geGiftEntryFormApp/__tests__/geGiftEntryFormApp.test.js

```diff
@@ -0,0 +1,70 @@
+import { createElement } from 'lwc';
+import GeGiftEntryFormApp from 'c/geGiftEntryFormApp';
+import { getRecord } from "@salesforce/sfdx-lwc-jest/src/lightning-stubs/uiRecordApi/uiRecordApi";
+import getGiftBatchTotalsBy from '@salesforce/apex/GE_GiftEntryController.getGiftBatchTotalsBy';
+const mockGetRecord = require('./data/getRecord.json');
+const PROCESSING_BATCH_MESSAGE = 'c.geProcessingBatch';
+const batchTotalsProcessingGifts = require('./data/batchTotalsProcessingGifts.json');
+
+const createGeGiftEntryFormApp = () => {
+    return createElement('c-ge-gift-entry-form-app',
+        {is: GeGiftEntryFormApp}
+    );
+}
+
+jest.mock(
+    '@salesforce/apex/GE_GiftEntryController.getGiftBatchTotalsBy',
+    () => {
+        return {
+            default: jest.fn(),
+        };
+    },
+    { virtual: true }
+);
+
+describe('c-ge-gift-entry-form-app', () => {
+
+    afterEach(  () => {
+       clearDOM();
+       jest.clearAllMocks();
+    });
+
+    it('should render processing batch spinner if batch is still processing', async  () => {
```

Comment:
This looks great!  What would you think about adding tests to verify that the spinner starts when it is supposed to (process batch clicked and Batch has pmts to process), and stops when it is supposed to (last Gift is done processing)?

Reply:
I would love to add the tests to cover the spinner not showing after processing, but it requires me to mock all the child components(`geFormRenderer` and `geBatchGiftEntryTable`) in `geGiftEntryFormApp` which is a lot to do right now. So I thought I thought I get the ball rolling with this one.